script/deploy/zksync/DeployLiFiDEXAggregator.s.sol

```diff
@@ -30,6 +30,14 @@ contract DeployScript is DeployScriptBase {
 
         // the original RouteProcessor4.sol is also deployed with address(0) for _bentoBox
 
-        return abi.encode(address(0), priviledgedUsers);
+        // extract withdrawWallet address
+        address withdrawWalletAddress = json.readAddress(
+            ".withdrawWallet"
+        );
+        require(
```

Comment:
Why is this here?
This is unusual code. Normally we use the deployRequirements.json to define required parameters and the scripts should make sure that these parameters are available

Reply:
@0xDEnYO fixed